lib/util.js

```diff
@@ -109,6 +109,16 @@ module.exports.log = internals.consoleFn('log');
 module.exports.dir = internals.consoleFn('dir');
 
 
+module.exports.hash = function (values) {
```

Comment:
My point is that this is an internal utility function that only gets used one way... so you can simplify it.